Formula/emqttd.rb

```diff
@@ -0,0 +1,45 @@
+class Emqttd < Formula
+  desc "Erlang MQTT Broker"
+  homepage "http://emqtt.io"
+  url "http://emqtt.io/emqttd-macosx-2.1.0.zip"
+  sha256 "c3d213c1085e60ad767fe277635d369620903f041fadc0fb8c7630277b5d3af9"
```

Comment:
I do not know why there will be SHA mismatch:
Http://emqtt.io/emqttd-macosx-2.1.0.zip package sha256 is c3d213c1085e60ad767fe277635d369620903f041fadc0fb8c7630277b5d3af9
I took the machine new mac
1. cp emqttd.rb  /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/
2. brew install emqttd
3. The installation was successful